components/brave_ads/core/internal/ads_feature.cc

```diff
@@ -18,7 +18,7 @@ bool ShouldLaunchAsInProcessService() {
 
 BASE_FEATURE(kShouldAlwaysRunBraveAdsServiceFeature,
              "ShouldAlwaysRunBraveAdsService",
-             base::FEATURE_DISABLED_BY_DEFAULT);
+             base::FEATURE_ENABLED_BY_DEFAULT);
```

Comment:
Unfortunately, this is not possible at the moment as we are investigating the cause of a Chromium test failure. It was reviewed and appears to be just a test issue. We will add this to our list of priorities. Thank you. cc @aseren